README.md

```diff
@@ -52,6 +52,55 @@ docker_mgmt:
 
 These values are used by the Ansible `git` task to perform the actual cloning.
 
+## Roles
+
+### deploy_site
+
+`deploy_site` is a base level role that is to be used for all sites within the
+`metacpan-docker` environments. The `site` variable is a required setting and
+must match the name of the service to be started, and the configuration values
+in the `var/settings.yml` file.
+
+The `deploy_site` role is comprised of 3 parts:
+
+1. configure
+2. compose_pull
+3. compose_up
+
+#### configure
+
+The configure phase integrates the values for the site in `vars/settings.yml`
+into the `{{ site }}/enviornment.json` file. Site settings keys are used to
```

Comment:
I think this should be `environment.json`

Reply:
Oops. Thank you.